pkg/controller/certificatesigningrequest/helpers.go

```diff
@@ -73,7 +73,7 @@ func isNodeServingCert(csr *capi.CertificateSigningRequest, x509cr *x509.Certifi
 		klog.Warningf("Org does not match: %s", x509cr.Subject.Organization)
 		return false
 	}
-	if (len(x509cr.DNSNames) < 1) || (len(x509cr.IPAddresses) < 1) {
+	if (len(x509cr.DNSNames) < 1) && (len(x509cr.IPAddresses) < 1) {
 		return false
```

Comment:
As defined in the original issue #35 maybe we should also log here something why we're rejecting the CSR. I.e. something like `No DNS or IP addresses given in CSR`

Reply:
Added!